app/components/services-section/services-card/index.jsx

```diff
@@ -0,0 +1,95 @@
+'use client'
+
+import { icon } from '@fortawesome/fontawesome-svg-core';
+import React, { useEffect } from 'react';
+
+
+function Card({ id, title, img, bgColor, service_detail }) {
+
+    //esta funcion cambia de estado de isopen , osea cambia el estado de false a true , se usa despues con onclick. El operador lógico !
+    //devuelve true cuando la expresión setAbierta da como resultado false 
+
+
+    // Recordar : El operador lógico AND (&&) será true para un conjunto de operandos booleanos,solo si todos los operandos son true.
+    //         sino sera false.
+
+
+    // aca se entiende que isOpen es la card cardAbierta pero que en su estado inicial es false (osea que no esta isOpen),
+    // mientras que si se usa el operador ! este estado cambia, osea es el opuesto
+
+
+
+    useEffect(() => {
```

Comment:
which is the propousal of this implementation?

Reply:
se intenta emular el hover de css para dispositivos tactiles

Reply:
excelente! lo dejemos para otra tarea dado que es dificil de emular su funcionamiento y el codigo pareciera no hacer lo esperado